packages/components/src/DataTable/DataTable.css

```diff
@@ -36,6 +36,20 @@
   padding: var(--space-base);
 }
 
+.table tfoot {
+  position: sticky;
+  z-index: 1;
+  width: 100%;
+  box-shadow: var(--shadow-base);
+  border-top: var(--border-thick) solid var(--color-border);
+  background-color: var(--color-white);
```

Comment:
```suggestion
  background-color: var(--color-surface);
```

Reply:
@chris-at-jobber  Thanks! Just pushed this change :)